### PR TITLE
Feature/first release

### PR DESCRIPTION
**Pull request by emielver**

### CODE REVIEW

File: custom_users/packages.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+packages:
+  - git: "https://github.com/snowplow/dbt-snowplow-utils.git"
+    revision: "release/snowplow-utils/0.15.0"
```
Comment (rlh1994) [inline custom_users/packages.yml]: Is this going to change when it's released?

File: custom_sessions/models/base/snowplow_base_events_this_run.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#}
+
+{%- set lower_limit, upper_limit = snowplow_utils.return_limits_from_model(ref(var('snowplow__base_sessions', 'snowplow_base_sessions_this_run')),
+                                                                            'start_tstamp',
+                                                                            'end_tstamp') %}
+
+{% set base_events_query = snowplow_utils.base_create_snowplow_events_this_run(
+    sessions_this_run_table=var('snowplow__base_sessions', 'snowplow_base_sessions_this_run'),
+    session_identifiers=var('snowplow__session_identifiers', '[{"schema": "contexts_com_snowplowanalytics_session_identifier_2_0_0", "field": "session_identifier", "prefix": "si_t", "alias": "sito"}, {"schema": "contexts_com_snowplowanalytics_session_identifier_1_0_0", "field" : "session_id", "prefix" : "si_o", "alias": "sido"}]'),
+    session_timestamp=var('snowplow__session_timestamp', 'collector_tstamp'),
```
Comment (rlh1994) [inline custom_sessions/models/base/snowplow_base_events_this_run.sql]: I would maybe not add fallbacks for the variables here, as it is a little confusing/duplication from the project yaml?

File: custom_users/packages.yml (hunk 1 above)
Comment (emielver) [reply custom_users/packages.yml]: We can change it now, since it works as tested

File: custom_base/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Usage
+
+In order to include custom entities and SDEs , we set the `snowplow__entities_or_sdes` to have the following value `[{"schema" : "contexts_com_snowplowanalytics_custom_entity_1_0_0", "prefix": "custom", "single_entity": true}]`. This pulls the `contexts_com_snowplowanalytics_custom_entity_1_0_0` context into the `snowplow_base_events_this_run` table, which is useful if you have a federated table design where the entities are stored in tables separate from the `atomic.events` table. This is generally only the case with Redshift and Postgres setups. We also add custom SQL into the `base_events_this_run` model by specifying the `snowplow__custom_sql` to be `contexts_com_snowplowanalytics_custom_entity_1_0_0[0].contents::TEXT as custom_contents`, which will then include this SQL in our `snowplow_base_events_this_run` model, to be able to be used by subsequent models.
+
```
Comment (rlh1994) [inline custom_base/README.md]: does this custom_sql only work for one warehouse?

**Review by rlh1994 — COMMENTED** [review]
A few comments

**Review by rlh1994 — APPROVED** [review]
https://github.com/snowplow-incubator/dbt-example-project/assets/8260415/77795315-30e4-4eb4-91ba-d037c186600a

**Pull request merged**